frontend/src/components/maps/leaflet/InfoSlideOut/ThreeColumnItem.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Col, ListGroup, Row } from 'react-bootstrap';
+import { Label } from 'components/common/Label';
+
+const InnerRow = styled(Row)`
+  margin: 0px;
+  width: 300px;
```

Comment:
assuming the outer container has a fixed width, you should be able to use col-md-x to size columns based on a percentage of the overall row. This is a bit less brittle then specifying multiple fixed widths, but may not always be an option depending on the situation.

What I'm trying to say is, please use col-md-x where possible. if not, feel free to use fixed widths.

Reply:
can you do that with styled components?

Reply:
I played around with it, I couldn't seem to get it to work